eiger.cscs.ch/default

```diff
@@ -0,0 +1 @@
+normal
```

Comment:
I would suggest calling it `mr0`, or `multicore-mr0`.

Reply:
Would you also change the computer labels then?

Reply:
yes, I suggest `eiger-mr0`. The thing is, if you would use another project, you would have to re-create the computer anyways.